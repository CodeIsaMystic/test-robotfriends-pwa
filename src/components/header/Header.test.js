import React from 'react';
import { create } from "react-test-renderer";


import Header from './Header';
import CounterButton from '../counter-button/CounterButton';



/*   Entering all tests in describe   */
describe('header', () => {

  const header = create(<Header />);


  it('to render Header Component with react test renderer', () => {

    expect(header.toJSON()).toMatchSnapshot();
  });

});