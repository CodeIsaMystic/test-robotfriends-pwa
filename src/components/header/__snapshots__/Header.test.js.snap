// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`header to render Header Component with react test renderer 1`] = `
<div>
  <h1
    className="f1"
  >
    RobotFriends
  </h1>
  <button
    id="counter"
    onClick={[Function]}
  >
    Count: 
    0
  </button>
</div>
`;
